packages/hoprd/src/index.ts

```diff
@@ -135,6 +135,11 @@ const argv = yargs(process.argv.slice(2))
     describe: 'A private key to be used for your HOPR node',
     default: undefined
   })
+  .option('provider', {
+    string: true,
+    describe: 'A custom RPC provider to be used for your HOPR node to connect to blockchain',
+    default: 'http://127.0.0.1:8535/'
```

Comment:
Default should be the one defined in our environment.

Reply:
if `--provider` is not specified, default from environment files will be used

Reply:
Removed the default, because the default value should be populated per environment.